internal/driver/webhtml.go

```diff
@@ -511,6 +615,11 @@ function dotviewer(nodes) {
     return null
   }
 
+  // convert a string to a regexp that matches that string.
+  function quotemeta(str) {
+    return str.replace(/([\\\.?+*\[\](){}])/g, '\\$1')
```

Comment:
Thanks. However, my profile has one function with a very long name, so the inline column gets pushed to the right so much that I cannot see it anymore. I could scroll-right to see it, but now it is so far away that it is hard to see which function is which.

Could we place the Inline column to the left of the Name column so that it always fits, or cap the width of the Name column?